### PR TITLE
Session management

### PR DESCRIPTION
**Pull request by Julianhm9612**
## Summary

- Added NavBar component
- Added Home screen
- Added redirects to Home
- Configured session management with localStorage

## Screenshots

Session management

## Trello Card

https://trello.com/c/VkXa68Vk/15-session-management-logout

### CODE REVIEW

File: src/components/NavBar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+function NavBar() {
+  const navigate = useNavigate();
+
+  const logout = () => {
+    logoutSession();
+    navigate('/');
+  };
+
+  return (
+    <div className={styles.navbar}>
```
Comment (ebedoyaalzate) [inline src/components/NavBar/index.tsx]: this div is unnecessary

File: src/components/NavBar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      <nav className={styles.nav}>
+        <img src={logo} className={styles.logo} alt="Logo original de wolox" />
+        <button className={styles.logout} onClick={logout}>
+          Logout
```
Comment (ebedoyaalzate) [inline src/components/NavBar/index.tsx]: add i18n

File: src/components/NavBar/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+.navbar {
+    background-color: $white;
+    border-top: 6px solid $vivid-cerulean;
+    box-shadow: 0 4px 10px 0 $black-shadow;
+    display: flex;
+    height: 70px;
+    justify-content: center;
+    position: fixed;
+    top: 0;
```
Comment (ebedoyaalzate) [inline src/components/NavBar/styles.module.scss]: two spaces of tab

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

File: src/components/NavBar/index.tsx (hunk 1 above)
Comment (Julianhm9612) [reply src/components/NavBar/index.tsx]: Has been deleted and improved :godmode:

File: src/components/NavBar/index.tsx (hunk 2 above)
Comment (Julianhm9612) [reply src/components/NavBar/index.tsx]: Has been added :godmode:

File: src/components/NavBar/styles.module.scss (hunk 3 above)
Comment (Julianhm9612) [reply src/components/NavBar/styles.module.scss]: Has been solved :godmode:

File: src/components/NavBar/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  height: auto;
+  width: 212px;
+}
+
+.logout {
+  color: $black;
+  font-family: Roboto;
+  font-size: 18px;
+  line-height: 21px;
+}
```
Comment (ebedoyaalzate) [inline src/components/NavBar/styles.module.scss]: empty line

File: src/screens/Home/index.tsx [hunk 5]
```diff
@@ -3,14 +3,12 @@ import i18next from 'i18next';
...
+import NavBar from 'components/NavBar';
 
 function Home() {
   return (
-    <div className={styles.app}>
-      <header className={styles.appHeader}>
-        <img src={logo} className={styles.appLogo} alt={i18next.t('Home:logoAlt') as string} />
-        <p className={styles.text}>{i18next.t('Home:welcome')}</p>
-      </header>
+    <div className={styles.home}>
```
Comment (ebedoyaalzate) [inline src/screens/Home/index.tsx:10]: do you need this div?

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

**Review by ebedoyaalzate — APPROVED** [review]

File: src/components/NavBar/styles.module.scss (hunk 4 above)
Comment (Julianhm9612) [reply src/components/NavBar/styles.module.scss]: My mistake! Has been solved :godmode:

**Pull request merged**